easybuild/easyblocks/generic/tarball.py

```diff
@@ -55,15 +55,17 @@ def build_step(self):
         """
         pass
 
-    def install_step(self):
+    def install_step(self, src=None):
```

Comment:
I don't think that makes a lot of sense? usually, no 'source' is involved, and it's just a command to run